rpcs3/Emu/RSX/Overlays/overlay_fonts.cpp

```diff
@@ -136,7 +136,7 @@ namespace rsx
 				result.font_names.emplace_back("SCE-PS3-SR-R-JPN.TTF");
 
 				// Known system font as last fallback
-				result.font_names.emplace_back("Yu Gothic.ttf");
+				result.font_names.emplace_back("YuGothR.ttc");
```

Comment:
Not sure outside of Win10 (as this wwas tested on), needs checking on other operating systems. Same for the rest.

Reply:
I'll check on linux in a few hours, we can use multiple fallbacks anyway so its not too big of a deal.

Reply:
Added both to ease up merging, can be investigated deeply after this pr and having both does not do any harm.